### PR TITLE
Update XRootD Standalone and other XRootD docs to OSG 23 (SOFTWARE-5820)

### PR DESCRIPTION
**Pull request by matyasselmeci**
This updates the XRootD Standalone doc for OSG 23, and also some related docs (XRootD Storage Element, XRootD Authorization) since the changes were similar.

### CODE REVIEW

**Comment by matyasselmeci** [comment]
I reverted the changes to install-storage-element.md. I'll make a separate PR to de-list the page.

File: docs/data/xrootd/install-standalone.md [hunk 1]
```diff
@@ -118,6 +108,8 @@ To set up XRootD in multi-user mode, install the `xrootd-multiuser` package:
 root@xrootd-standalone # yum install xrootd-multiuser
 ```
 
+If you are using XRootD-Multiuser with a VOMS FQAN, you need XRootD 5.5.0 or greater.
```
Comment (brianhlin) [inline docs/data/xrootd/install-standalone.md:111]: This is a little hidden, let's turn it into an admonition. Pick your favorite one: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types

File: docs/data/xrootd/xrootd-authorization.md [hunk 2]
```diff
@@ -356,6 +350,11 @@ pair, `xrootd-client`, and `voms-clients-cpp` installed:
     If your transfer does not succeed, re-run `xrdcp`  with `--debug 2` for more information.
 
 
+Updating to OSG 23
+------------------
+
+There are no manual steps necessary for authentication to work when upgrading from OSG 3.6 to OSG 23.
+If you are upgrading from an earlier version, see the [updating to OSG 3.6](#updating-to-osg-3-6) section below.
```
Comment (brianhlin) [inline docs/data/xrootd/xrootd-authorization.md:357]:
```suggestion
If you are upgrading from an earlier release series, see the [updating to OSG 3.6](#updating-to-osg-3-6) section below.
```

**Review by brianhlin — APPROVED** [review]
Mostly LGTM. Pre-approving

**Comment by matyasselmeci** [comment]
This PR is not picking up my latest changes so I'm going to close it and open a new one.

**Pull request closed without merging**